contracts/strategies/traderjoe/TraderJoe.sol

```diff
@@ -4,17 +4,16 @@ pragma solidity ^0.8.0;
 import "./TraderJoeStorageLib.sol";
 import "../../common/bases/StrategyOwnablePausableBaseUpgradeable.sol";
 import "../../common/libraries/SwapServiceLib.sol";
-import "../../dependencies/traderjoe/ITraderJoeMasterChef.sol";
-import "../../dependencies/traderjoe/ITraderJoeRouter.sol";
-import "../../dependencies/traderjoe/ITraderJoePair.sol";
+import "../../dependencies/traderjoe/ITraderJoeLBPair.sol";
+import "../../dependencies/traderjoe/ITraderJoeLBRouter.sol";
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 
 contract TraderJoe is UUPSUpgradeable, StrategyOwnablePausableBaseUpgradeable {
     using SafeERC20Upgradeable for IInvestmentToken;
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
-    error InvalidTraderJoeLpToken();
+    error InvalidTraderJoeLBPair();
 
     // solhint-disable-next-line const-name-snakecase
     string public constant trackingName =
```

Comment:
can we change the version number?

Reply:
To what? I left that unchanged so that you can decide :)

Reply:
it is a bigger change (but not breaking), so how about we bump up the middle version by 1?